Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -406,18 +405,50 @@ function New-LDAPSIdentitySource {
     else {
         Write-Host "No existing external identity sources found."
     }
-
     $Password = $Credential.GetNetworkCredential().Password
-    $DestinationFileArray = Get-Certificates -SSLCertificatesSasUrl $SSLCertificatesSasUrl -ErrorAction Stop
-    [System.Array]$Certificates = 
-        foreach($CertFile in $DestinationFileArray) {
+    $DestinationFileArray = @()
+    if ($PSBoundParameters.ContainsKey('SSLCertificatesSasUrl')) {
+        $DestinationFileArray = Get-Certificates -SSLCertificatesSasUrl $SSLCertificatesSasUrl -ErrorAction Stop
+    } else {
+        $exportFolder = "$home/"
+        $remoteComputers = ,$PrimaryUrl
+        if ($PSBoundParameters.ContainsKey('SecondaryUrl')) { $remoteComputers += $SecondaryUrl }
+        
+        foreach ($computerUrl in $remoteComputers) {
+            if ($computerUrl.IndexOf('ldaps://') -ne 0) { Write-Error "Incorrect Url format entered: " + $computerUrl -ErrorAction Stop}
+
+            $TrimedUrl = $computerUrl.substring('ldaps://'.Length)
+            $Command = 'echo "1" | openssl s_client -connect ' + $TrimedUrl + ' -showcerts' | out-string
```

Comment:
is this supposed to be a "1"? What's that for?

Reply:
yeah, return 1 after openssl to avoid the idle for inputs.